package.json

```diff
@@ -3,11 +3,14 @@
   "version": "0.0.1",
   "description": "Application used to enable voting on stories in a asynchronous way",
   "main": "bin/server.js",
-  "engines" : { "node" : ">=6.0.0" },
+  "engines": {
+    "node": ">=6.0.0"
+  },
   "scripts": {
     "test": "cross-env NODE_ENV=test mocha -R spec test/*/*.spec.js",
     "start": "node bin/server.js",
     "debug": "node debug bin/server.js",
+    "lint": "./node_modules/.bin/eslint src/ --color",
```

Comment:
`npm run` uses the `node_modules/.bin` path by default.

`"lint": "eslint src/ --color",`